tests/src/utils/test_utils.py

```diff
@@ -165,3 +167,18 @@ def test_should_load_packed_bytes(self, cairo_run):
         packed_bytes = pack_calldata(bytes)
         output = cairo_run("test__load_packed_bytes", data=packed_bytes)
         assert output == list(bytes)
+
+    def test_should_raise_zellic_issue_1283_load_packed_bytes(
+        self, cairo_program, cairo_run
+    ):
+        bytes = random.randbytes(100)
+        packed_bytes = pack_calldata(bytes)
+        with (
+            patch_hint(
+                cairo_program,
+                "memory[ids.output] = res = (int(ids.value) % PRIME) % ids.base\nassert res < ids.bound, f'split_int(): Limb {res} is out of range.'",
+                "memory[ids.output] = res = 0x12\nassert res < ids.bound, f'split_int(): Limb {res} is out of range.'",
+            ),
+        ):
+            with cairo_error(message="Value is not empty"):
+                cairo_run("test__load_packed_bytes", data=packed_bytes)
```

Comment:
```suggestion
        with (
            patch_hint(
                cairo_program,
                "memory[ids.output] = res = (int(ids.value) % PRIME) % ids.base\nassert res < ids.bound, f'split_int(): Limb {res} is out of range.'",
                "memory[ids.output] = res = 0x12\nassert res < ids.bound, f'split_int(): Limb {res} is out of range.'",
            ),
            cairo_error(message="Value is not empty"),
        ):
```

Reply:
(using one single `with` if it works)

Reply:
simplifying the patch to see better what we do, we don't need the assert here which are made only for dev purposes